docs/reference/components/amqp.cue

```diff
@@ -0,0 +1,116 @@
+package metadata
+
+components: _amqp: {
+	features: {
+		collect: from: {
```

Comment:
I believe this `collect` feature should only be on the source and the `send` feature only on the sink.